Lib/gftools/builder/operations/instantiateUfo.py

```diff
@@ -59,6 +59,8 @@ def targets(self):
     def variables(self):
         vars = super().variables
         vars["args"] += " --ufo-structure=json "
+        if self.original["expandFeaturesToInstances"]:
```

Comment:
Any reason why you're using `self.original` here? It is breaking the tests since `expandFeaturesToInstances` isn't in `self.original`. If I change it to `vars` it seems to be working but I could be wrong.

Personally, I would use a dict's `get` method if I cannot guarantee that a key exists. It also allows you to set a value incase it doesn't exist so in your case `self.original.get("expandFeaturesToInstances", False)` will return False if the key doesn't exist, or it will simply return the value if it does.


`self.original.get("expandFeaturesToInstances", False)` is what

Reply:
First, thank you for that tip on `.get()` – that is news to me, but really helpful to learn!

> Any reason why you're using self.original here?

As far as I can remember, I think I just tried to use a similar format as what is a few lines below:

https://github.com/googlefonts/gftools/blob/cc8617e1a55eeaeb25a0e8074abe67c0657744c3/Lib/gftools/builder/operations/instantiateUfo.py#L69

...and when it seemed to work, I thought I had solved it.

My impression was that `self.original` is the `GOOGLEFONTS_SCHEMA`, but I now realize that, even if that is true, there are other possible schemas.